integration-tests/api/__tests__/admin/sales-channels.js

```diff
@@ -267,16 +276,239 @@ describe("sales channels", () => {
       expect(response.data).toEqual({
         id: expect.any(String),
         object: "sales-channel",
-        deleted: true
+        deleted: true,
       })
 
       deletedSalesChannel = await dbConnection.manager.findOne(SalesChannel, {
         where: { id: salesChannel.id },
-        withDeleted: true
+        withDeleted: true,
       })
 
       expect(deletedSalesChannel.id).toEqual(salesChannel.id)
       expect(deletedSalesChannel.deleted_at).not.toEqual(null)
     })
   })
+
+  describe("GET /admin/orders/:id", () => {
```

Comment:
question:  I am not sure that those tests cases should belong to the integration sales channels. Do you think that we should move them closer to there tests suite?